src/content/docs/query-your-data/explore-query-data/dashboards/dashboard-template-variables.mdx

```diff
@@ -2,19 +2,21 @@
 title: "Template variables: dynamically filter dashboards"
 tags:
   - Dashboards
+  - Template variables
 metaDescription: 'For New Relic custom dashboards: use template variables to allow your dashboard users to dynamically filter your dashboard views.'
 redirects:
 freshnessValidatedDate: never
 ---
 
-For custom dashboards, you can use <DNT>**template variables**</DNT> to dynamically filter charts and other widgets. Template variables help make your dashboards more useful, and help you to more easily create dashboards you can reuse for different use cases.
+For custom dashboards, you can use <DNT>**template variables**</DNT> to dynamically filter charts and other widgets. Template variables make your dashboards more useful and help you more easily create dashboards that you can reuse for different use cases.
```

Comment:
Alternate: Template variables make your dashboards more useful by allowing you to create reusable dashboards for different use cases.
or
You can also use Template variables to create reusable dashboards for different use cases.